ecommerce/serializers.py

```diff
@@ -361,6 +363,31 @@ def get_data_consents(self, instance):
         data_consents = get_or_create_data_consent_users(instance)
         return DataConsentUserSerializer(instance=data_consents, many=True).data
 
+    def get_tax_info(self, _):
+        """Get the tax information for the current basket"""
+        request = self.context.get("request", None)
+        tax_info = {
+            "country_code": "",
+            "tax_rate": 0,
+            "tax_rate_name": "",
+        }
+
+        if request and hasattr(request, "user"):
+            country_code = determine_visitor_country(request)
+            if country_code is not None:
+                try:
+                    rate = TaxRate.objects.get(country_code=country_code)
+                except TaxRate.DoesNotExist:
+                    return tax_info
+
+                tax_info["country_code"] = country_code
+                tax_info["tax_rate"] = rate.tax_rate
+                tax_info["tax_rate_name"] = rate.tax_rate_name
```

Comment:
I simplified this to just run `__dict__` on the resulting TaxRate object - this will change the data out here slightly but this key is also new so this shouldn't be a big deal.